README.md

```diff
@@ -61,14 +49,17 @@ When defined, the liquidity adapter is also used to forward deposited funds.
 
 A typical liquidity adapter would allow deposits/withdrawals to go through a very liquid Market v1.
 
-### Non-custodial guarantees
+### Timelocks
 
-Non-custodial guarantees come from [in-kind redemptions](#in-kind-redemptions-with-forcedeallocate) and [timelocks](#curator-timelocks).
-These mechanisms allow users to withdraw their assets before any critical configuration change takes effect.
+Curator configuration changes are all timelockable (except `decreaseAbsoluteCap` and ), meaning that doing an action requires submitting it first, and only when the timelock has passed it can be executed (by anyone).
+This is useful notably to the [non-custodial guarantees](#non-custodial-guarantees), but also in general if a curator wants to give guarantees about some configurations.
+
+In particular, a configuration can be *abdicated*, meaning that it won't be able to be set anymore, by setting the timelock to type(uint256).max and making sure that pendingCount for this selector is zero.
+Thus, increaseTimelock should be used carefully, because decreaseTimelock is timelocked with the timelock itself.
```

Comment:
```suggestion
Thus, increaseTimelock should be used carefully, because decreasingTimelock is function-dependent: decreasing the timelock of a function is timelocked by the timelock of the function itself.
```